front/src/common/Grid.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cx from 'classnames';
+import './Grid.scss';
+
+type GridProps = {
+  data: string[];
+  filter?: string;
+  extraClass?: string;
+  updateData: (data: string) => void;
+  closeModal?: () => void;
+};
+
+function isFiltered(item: string, input: string): boolean {
```

Comment:
I find it clearer to keep the name filter here :) 
```suggestion
function isFiltered(item: string, filter: string): boolean {
```

Reply:
done :)